go.mod

```diff
@@ -42,7 +42,7 @@ require (
 	github.com/go-test/deep v1.0.2
```

Comment:
for thrift error, as master branch, we need to put below before require block

replace git.apache.org/thrift.git => github.com/apache/thrift v0.12.0